src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -1178,6 +1187,13 @@ ParamBlockDesc2 g_param_block_desc(
         p_accessor, &g_pblock_accessor,
     p_end,
 
+    ParamIdNoiseSeed, L"noise_seed", TYPE_INT, P_TRANSIENT, 0,
+        p_ui, ParamMapIdImageSampling, TYPE_SPINNER, EDITTYPE_INT, IDC_TEXT_NOISE_SEED, IDC_SPINNER_NOISE_SEED, SPIN_AUTOSCALE,
+        p_default, 0,
+        p_range, 0, 10000000,
```

Comment:
If possible, the range should be [0, 4294967295], or if that's not possible because the underlying type is a 32-bit _signed_ integer, then it should be [-2147483648, 2147483647].